Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import DequeModule
+import Foundation
+import GRPC
+import SwiftProtobuf
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+public class ReflectionService: CallHandlerProvider {
```

Comment:
This should be a `struct`

Reply:
It also needs to conform to `Sendable`

Reply:
But CallHandlerProvider is a class-only protocol as far as I can tell (it conforms to AnyObject)